app/src/main/java/emu/skyline/preference/CustomEditTextPreference.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * SPDX-License-Identifier: MPL-2.0
+ * Copyright © 2020 Skyline Team and Contributors (https://github.com/skyline-emu/)
+ */
+
+package emu.skyline.preference
+
+import android.content.Context
+import android.text.InputFilter
+import android.text.InputFilter.LengthFilter
+import android.util.AttributeSet
+import androidx.preference.EditTextPreference
+import emu.skyline.R
+
+/**
+ * [EditTextPreference] lacks the feature to set the automatically value as summary.
+ * This class adds this missing thing. Also added useful attributes.
```

Comment:
`This class adapts [EditTextPreference] so that it supports setting the value as the summary automatically.`